pocs/observatory.py

```diff
@@ -53,12 +50,8 @@ def __init__(self, cameras=None, scheduler=None, dome=None, *args, **kwargs):
 
         # TODO(jamessynge): Discuss with Wilfred the serial port validation behavior
         # here compared to that for the mount.
-        self.logger.info('\tSetting up dome.')
```

Comment:
Let's keep these log statements but change to "Addng dome to Observatory" or something similar.